src/posts/experimental-static-build.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Scaling Astro Sites to Thousands of Pages'
+description: 'A new experimental flag in Astro build enables building sites with tens of thousands of pages.'
+publishDate: 'January 25, 2022'
+authors:
+  - matthew
+lang: 'en'
+---
+
+Astro's about to get a lot faster with a new build system intended to scale to tens, or even hundreds, of thousands of pages. If you hang out in our [Discord](https://astro.build/chat) or pay attention to recent releases you might have seen a lot of discussion about a "static build". This is our new implementation of `astro build` that does 2 things:
+
+- Improves build times by up to 75%.
+- Sets Astro up to support server-side rendering (SSR) in the near future.
```

Comment:
I'm not sure if most users care about this! Maybe as a point in the paragraph below (since you talk about server-side rendering there) but the main reason that a user would care about this is the speed and scaling improvements.

I also wouldn't shy away from comparing this to how Astro currently works. It can really hit home why this is a big deal, vs. "huh, I guess I thought that was how Astro already works?"

Something like:

```
Previously, Astro would build every page to HTML and then optimize each of those pages. This worked fine for small sites, but became a problem as sites grew larger.
```

Reply:
Hm, I'm not sure I agree (or maybe I don't understand). You're saying that users don't care about this setting Astro up for SSR but they *do* care about internal implementation details of the build? I sort of see it the opposite. I can see a technical article that explains the *how* of it being faster, but I wouldn't think that a typical dev cares about that. But I would expect them to care about SSR.

Reply:
Fair enough! You can make the final call here. It's a trap that I always fall into so I'm extra cautious: does an external audience care about the big, long-term strategic reason for the improvement vs. the improvement itself (speed & scaling)? I think it's great to mention somewhere, but my comment was mainly directed at the fact that it's a high-level callout here.

Reply:
Also now that i re-read this, is "improve build times by up to 75%" a good reflection of the improvement being made here? The title mentions scaling, but the call out here is overall speed. Maybe rewrite to better connect to the point made in the title? Something like: "Improves build times, especially for large site"

Reply:
I switched this point to be able lowering memory usage instead since that aligns with the immediate value.